### DISCUSSION TITLE
Migrating to React Native 0.60 and Firebase v6, missing firebase.notifications

### DISCUSSION THREAD

**Discussion by Simon-44** [invertase/react-native-firebase#4384] [Q&A] [answered]
Hello,

I'm migrating to React Native 0.60 for a few days and it seems my v5.5.5 of Firebase is not compatible anymore because it's making an import from android.support which is not compatible with AndroidX.

So I wanted to upgrade Firebase to the v6, but while following the tutorial (https://rnfirebase.io/migrating-to-v6) I realized the Notifications package is not in the updated packages of Firebase.

I'm kinda stuck and can't find anything online, like I'm the only one having troubles with Firebase v5 being obsolete for React Native 0.60 but Firebase v6 not coming with the Notifications package? Didn't find anything about migrating from Notifications to another package, so I feel like I'm missing something here.

Thanks.

**Comment by mikehardy** [comment, marked as answer]
You are unable to find anything online? :thinking: 

https://duckduckgo.com/?t=canonical&q=react-native-firebase+v6+notifications&atb=v204-1&ia=web

shows

https://github.com/invertase/react-native-firebase/issues/2566

says

```


Options are:

    react-native-firebase v5
    react-native-firebase v6 + react-native-push-notifications (or other 3rd party) for free
    react-native-firebase v6 + recently announced Invertase licensed product "Notifee" which is approaching delivery but not ready yet

If you have any questions please read this thread. If your question hasn't already been answered please feel to raise it as an issue on the new Notifee repo.
```

**Reply by Simon-44** [reply]
My bad, I read almost the whole thread except the original post. I guess using v5 is not an option anymore with RN 0.60.

Edit: oh okay there is the 5.6 for RN 0.60.

**Reply by mikehardy** [reply]
Worth mentioning (after a quick scan) that in the v5->v6 migration document it could use a mention perhaps. If you want to hit the edit button top right of the page and propose where you as a migrating user thinks it fits best, we could merge it so it was more obvious

Using v5 shouldn't be an option anymore just because it's so old it shouldn't be used, but of course it's open source, use it as long as it appears to work for you, just not that the underlying SDKs have moved on and - for instance - crashlytics won't work for you much longer as v5 hasn't migrated to Firebase Crashlytics and there are no plans to backport that migration to v6.
